StudentDriver/StudentDriver/Services/WebService.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using StudentDriver.Models;
+using Xamarin.Forms;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace StudentDriver.Services
+{
+    public static class WebService
+    {
+        private static string BaseUrl = "";
+
+        public static async Task<UserStats> GetStudentStats(int id)
+        {
+
+            var client = new HttpClient();
+            var requestUri = GenerateRequestUri(BaseUrl,"students", new Dictionary<string, string>() {{"userId", id.ToString()}});
+
+            var response = await client.GetAsync(requestUri);
+            var json = response.Content.ReadAsStringAsync().Result;
+            var userStats = JsonConvert.DeserializeObject<UserStats>(json);
+            return userStats;
+        }
+
+        public static async Task<bool> PostUnsyncDrivingSessions(List<UnsyncDrive> unsyncDrives)
+        {
+            var json = JsonConvert.SerializeObject(unsyncDrives);
+            var content = new StringContent(json, Encoding.UTF8, "application/json");
+
+            var client = new HttpClient();
+            var requestUri = GenerateRequestUri(BaseUrl,"drivingsessions", null);
+
+            var response = await client.PostAsync(requestUri, content);
+            return response.IsSuccessStatusCode;
+        }
+
+        public static async Task<StateReqs> GetStateReqs(string state)
```

Comment:
Change this a a method for pulling all states? Or add a method for pulling all state date?